### PR TITLE
refactor: reword and add styling to user emails

### PR DESCRIPTION
**Pull request by amy-corson-ibigroup**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [n/a] Any modified or new methods or classes have helpful JavaDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [n/a] The description lists any configuration setting(s) that differ from the default settings
- [ ] All tests and CI builds passing

### Description

- Updates language in user-facing trip notification emails to be more descriptive.
- Adds basic styling to template.

### CODE REVIEW

File: src/main/resources/templates/BaseEmail.ftl [hunk 1]
```diff
@@ -3,9 +3,25 @@
     emails that use this must implement a macro called `EmailBody` and the whole html email will
     eventually get rendered by the statement <@HtmlEmail/>.
 -->
+<style>
```
Comment (binh-dam-ibigroup) [inline src/main/resources/templates/BaseEmail.ftl:6]: Interesting. It looks like the `<style>` tag got inserted within a sparkpost-generated `<head>` tag.

**Review by binh-dam-ibigroup — APPROVED** [review]
Seems to work!

**Review by JymDyerIBI — APPROVED** [review]
Wording changes look good.

File: src/main/resources/templates/BaseEmail.ftl (hunk 1 above)
Comment (amy-corson-ibigroup) [reply src/main/resources/templates/BaseEmail.ftl:6]: Oh interesting, should I wrap the styling in a `<head>` tag here? Or is it fine to let Sparkpost handle that?

Comment (binh-dam-ibigroup) [reply src/main/resources/templates/BaseEmail.ftl:6]: I think it should be fine.

**Comment by binh-dam-ibigroup** [comment]
@amy-corson-ibigroup Could you update the snapshots?

**Pull request merged**